README.md

```diff
@@ -1,6 +1,6 @@
 <!--[![Travis Status](https://travis-ci.com/Pyomo/pyomo.svg?branch=master)](https://travis-ci.com/Pyomo/pyomo)-->
 [![Travis Status](https://img.shields.io/travis/com/Pyomo/pyomo/master?logo=travis)](https://travis-ci.com/Pyomo/pyomo)
-[![Appveyor Status](https://ci.appveyor.com/api/projects/status/km08tbkv05ik14n9/branch/master?svg=true)](https://ci.appveyor.com/project/WilliamHart/pyomo/branch/master)
+[![](https://github.com/Pyomo/pyomo/workflows/GitHub%20CI/badge.svg?event=push)](https://github.com/Pyomo/pyomo/actions)
```

Comment:
Three things:
1. Please put "GitHub Actions Status" in the alternative text field (the initial `![]`)
2. Can you make the link point to"https://github.com/Pyomo/pyomo/actions?query=event%3Apush+workflow%3A%22GitHub+CI%22" (just the list of master builds)?
3. Since we are moving things around, and since GHA is the new primary testng vehicle, can you move the badge up a line (before Travis)?

Reply:
Done!